src/model_diagnostics/scoring/scoring.py

```diff
@@ -318,11 +318,11 @@ def __init__(self) -> None:
 class LogLoss(_BaseScoringFunction):
     r"""Log loss.
 
-    The log loss is a strictly consistent scoring function for the mean for the
+    The log loss is a strictly consistent scoring function for the mean for
     observations and predictions in the range 0 to 1.
-    It is also referred to as Bernoulli deviance, Binomial log-likelihood, logistic
-    loss and binary cross-entropy.
-    It's minimal function value is zero.
+    It is also referred to as (half the) Bernoulli deviance, 
```

Comment:
```suggestion
    It is also referred to as (half the) Bernoulli deviance,
```
This should fix the linter (ruff) CI error.